netlify.toml

```diff
@@ -6,4 +6,6 @@
   # presently wrong.  I would encourage removing this eventually, but for now,
   # we'll check to see if `/bin/false` returns true (it won't) as a gauge.
   ignore  = "/bin/false"
-
+[build.environment]
+  NODE_VERSION = "14"
+  NPM_VERSION  = "7"
```

Comment:
I will trust you that this works; I see that the docs explicitly say that NODE_VERSION can be a single number and hope that the same is true for NPM_VERSION.

Reply:
Taken from the Netlify build logs from this branch:

```
3:28:00 PM: v14.16.1 is already installed.
3:28:00 PM: Now using node v14.16.1 (npm v6.14.12)
3:28:03 PM: Found npm version (6.14.12) that doesn't match expected (7)
Installing npm at version 7
3:28:10 PM: /opt/buildhome/.nvm/versions/node/v14.16.1/bin/npx -> /opt/buildhome/.nvm/versions/node/v14.16.1/lib/node_modules/npm/bin/npx-cli.js
3:28:10 PM: /opt/buildhome/.nvm/versions/node/v14.16.1/bin/npm -> /opt/buildhome/.nvm/versions/node/v14.16.1/lib/node_modules/npm/bin/npm-cli.js
3:28:10 PM: + npm@7.11.2
```